src/App.tsx

```diff
@@ -15,36 +17,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_ALPHABET = 'alphabet',
+  SORT_LENGTH = 'length',
+}
+
+const handleSort = (type: SortType | '', isReversed: boolean): string[] => {
```

Comment:
In this case better name getSortedGoods ( or something like that). 
Use the handle for events ( onClick, onChange, etc.)